### PR TITLE
[DOC] Add MWE to docstrings to *concatenate_columns and select_column

### PR DESCRIPTION
**Pull request by thatlittleboy**
# PR Description

Please describe the changes proposed in the pull request:

Added MWE for
- `concatenate_columns`
- `deconcatenate_column`
- `select_columns`

**This PR is related to #972.**

# Relevant Reviewers

Please tag maintainers to review.

- @ericmjl

### CODE REVIEW

**Comment by ericmjl** [comment]
🚀 Deployed on https://deploy-preview-994--pyjanitor.netlify.app

**Review by samukweku — APPROVED** [review]

File: janitor/functions/select_columns.py [hunk 1]
```diff
@@ -26,26 +27,18 @@ def select_columns(
...
-    Method-chaining example:
-
-    ```python
-        df = (pd.DataFrame(...)
-              .select_columns('a', 'b', 'col_*',
-              invert=True))
-    ```
+        >>> import pandas as pd
+        >>> import janitor
+        >>> df = pd.DataFrame({"col1": [1, 2], "foo": [3, 4], "col2": [5, 6]}); df
```
Comment (samukweku) [inline janitor/functions/select_columns.py]: can we shift the `df` after the `;` down to a new line. looks cleaner IMO

**Review by ericmjl — APPROVED** [review]
@samukweku please do the honors when everything is green!

File: janitor/functions/select_columns.py (hunk 1 above)
Comment (ericmjl) [reply janitor/functions/select_columns.py]: Appreciate the attention to detail here, @samukweku!

**Pull request merged**